scripts/orderlywebconfig.properties

```diff
@@ -1,4 +1,4 @@
 app.url=https://localhost/reports
 auth.fine_grained=false
-montagu.api_url=https://localhost/api/v1
+montagu.api_url=http://api:8080
```

Comment:
For a successful login orderly web needs to be able to talk to the api, which it couldn't before with this config.